tests/bluetooth/bsim_bt/bsim_test_audio/src/has_client_test.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifdef CONFIG_BT_HAS_CLIENT
+#include "bluetooth/audio/has.h"
+#include "common.h"
+
+extern enum bst_result_t bst_result;
+
+static volatile bool g_bt_init;
+static volatile bool g_is_connected;
+static volatile bool g_mtu_exchanged;
+static volatile bool g_has_discovered;
+static volatile bool g_active_preset_notified;
+static volatile bool g_preset_read;
```

Comment:
Consider using the CREATE_FLAG macros. Either should work, so this is fine too :)